server/sonar-server/src/main/java/org/sonar/server/component/ws/ComponentsWs.java

```diff
@@ -39,7 +39,7 @@ public ComponentsWs(AppAction appAction, SearchViewComponentsAction searchViewCo
   public void define(Context context) {
     NewController controller = context.createController("api/components")
       .setSince("4.2")
-      .setDescription("Components management");
+      .setDescription("Get information about a component (file, directory, project, ...) and its ancestors or children.");
```

Comment:
: ) you've got a point @ganncamp. My concern is that by being more specific we increase the likelihood of having a out-of-date documentation. I wasn't bothered by the Manage DOMAIN form personnaly. Here, I would have said « Manage components »